subprojects/model-core/src/main/java/org/gradle/api/internal/provider/proxies/package-info.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * A package for collection proxies for Property upgrades.
```

Comment:
💭 Instead of "proxy" I think "view" would be a better name for these. WDYT?

Reply:
Sure, we can use a view instead

Reply:
Renamed the package and also class names